snuba/subscriptions/worker.py

```diff
@@ -180,6 +180,18 @@ def callback_func(
                         },
                     )
 
+                    # Only log 1 in 1000 mistmatches to sentry
+                    if random.random() < 0.001:
```

Comment:
Maybe log more aggressively, right now we are snuplicating only 5% of the queries. 50% of the transactions do not match so you would log 1/1000 of 50% of 5%. We can afford a higher sampling rate.

Reply:
Also this logs both matches and non matches.

Reply:
Fixed